addons/web/static/tests/views/list_tests.js

```diff
@@ -1625,6 +1625,39 @@ QUnit.module('Views', {
         list.destroy();
     });
 
+    QUnit.test('selection box is disappeared after multi record save', async function (assert) {
+        assert.expect(5);
+
+        const list = await createView({
+            View: ListView,
+            model: 'foo',
+            data: this.data,
+            arch: '<tree multi_edit="1"><field name="foo"/><field name="bar"/></tree>',
+        });
+
+        await testUtils.nextTick();
+        assert.containsN(list, '.o_data_row', 4,
+            "there should be 4 records");
+        assert.containsNone(list.$('.o_cp_buttons'), '.o_list_selection_box',
+            "list selection box should not be displayed");
+
+        // select a record
+        await testUtils.dom.click(list.$('thead .o_list_record_selector input'));
+        assert.containsOnce(list.$('.o_cp_buttons'), '.o_list_selection_box',
+            "list selection box should be displayed");
+        assert.strictEqual(list.$('.o_list_selection_box').text().trim(), '4 selected',
```

Comment:
remove this assert (not in the scope of this test, and already tested in at least one other test)